OpenRA.Mods.RA/Scripting/Global/ReinforcementsGlobal.cs

```diff
@@ -110,7 +110,7 @@ public Actor[] Reinforce(Player owner, string[] actorTypes, CPos[] entryPath, in
 			"unless a custom exitFunc has been supplied. actionFunc will be called as " +
 			"actionFunc(Actor transport, Actor[] cargo). exitFunc will be called as exitFunc(Actor transport).")]
 		public LuaTable ReinforceWithTransport(Player owner, string actorType, string[] cargoTypes, CPos[] entryPath, CPos[] exitPath = null,
-			LuaFunction actionFunc = null, LuaFunction exitFunc = null)
+			int waitLand = 0, int waitReturn = 25, LuaFunction actionFunc = null, LuaFunction exitFunc = null)
```

Comment:
I think 0 and 25 as default values are fine. They can only ever be used if there are no action- or exitFunc, and this will mostly be the case with normal ground units or ships. Helicopters (which use the 15 and 50 ticks delays) will almost always have an action- and/or exitFunc, so setting the defaults to suit them doesn't make sense.